tendrl/notifier/notifier/notification/handlers/mail_handler.py

```diff
@@ -58,17 +58,38 @@ def get_config_help(self):
         return config_help
 
     def set_destinations(self):
-        # TODO(anmolbabu):User configuration will not just be destination email
-        # but it would also include user level subscriptions the capability
-        # underneath needs to be enhanced for that.
-        user_configs = []
+        # TODO(gowtham):Get user ids from indexes who enabled
+        # email notification, for now it is come from users path
+        user_ids = []
         try:
             users = central_store_util.read_key('/_tendrl/users')
             for user in users.leaves:
-                user_key = user.key
+                try:
+                    email_notifications = central_store_util.read_key(
+                        "%s/email_notifications" % user.key
+                    ).value
+                    if email_notifications == "true":
```

Comment:
You need to check whether user is enabled or not as well
Please confirm this @anivargi or @gnehapk whether this is part of milestone-2